src/ControlSystem/Tags/QueueTags.hpp

```diff
@@ -33,4 +35,59 @@ template <typename Frame>
 struct Horizon {
   using type = ::Strahlkorper<Frame>;
 };
+
+/// \ingroup ControlSystemGroup
+/// Holds a full strahlkorper from measurements for the excision surface
+template <typename Frame>
+struct ExcisionSurface {
+  using type = ::Strahlkorper<Frame>;
+};
+
+/// \ingroup ControlSystemGroup
+/// Holds a full strahlkorper from measurements for the time derivative of the
+/// apparent horizon surface
+template <typename Frame>
+struct TimeDerivApparentHorizon {
```

Comment:
[optional] We have a `dt` prefix tag for time derivatives, so you could use `::Tags::dt<Tags::Horizon<Frame>>` instead of this.